WebRoot/templates/prefs/Pages.template

```diff
@@ -2207,22 +2207,24 @@ NOTE: The tables inside of the filter rule sections MUST contain a tbody element
 <template id='prefs.Pages#GrantRightsDialog'>
 	<table role="presentation">
 		<tr id='${id}_name_row'>
-			<td class="Label" style="padding-right:10px"><$=ZmMsg.emailLabel$></td>
+			<td class="Label"><$=ZmMsg.emailLabel$></td>
+		</tr>
+		<tr>
 			<td id="${id}_name_cell">
 			</td>
 		</tr>
 		<tr>
-			<td class="Label">&nbsp;</td>
+			<td class="Label"></td>
```

Comment:
don't think we need this <td> as other rows are only having single <td> element. This row can be of same type.